src/library_nodefs.js

```diff
@@ -180,6 +180,7 @@ mergeInto(LibraryManager.library, {
         var newPath = PATH.join2(NODEFS.realPath(newDir), newName);
         try {
           fs.renameSync(oldPath, newPath);
+          oldNode.name = newName;
```

Comment:
please move this out of the `try`, as the try is only looking for FS exceptions.

Reply:
thanks for the review @kripken , fixed now.